### DISCUSSION TITLE
I can't verify that I am a teacher

### DISCUSSION THREAD

**Discussion by volkankoc** [community/community#147213] [GitHub Education] [answered]
### Hi everyone,

Hello
I am a teacher and have been using the teacher license on github for 2 years. Github notified me that I needed to renew this year. I am doing the same thing. I am currently at school and I am sharing a picture from the classroom with location data. But it says the location information is incorrect. I do not use VPN. Can you please solve my problem?

**Comment by samus-aran** [comment, marked as answer]
Hi @volkankoc 👋 

Thanks for raising this. First thing though we need to address on the application is you should complete your GitHub billing information with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before reapplying. If you have only a single legal name, enter it in both the first and last name fields.

Regards to the location we're confirming this for you on our end, we'll comment when we have an update on this.

I do want to ask if your school has verified school academic emails, if so perhaps you can make a request to update the school details. We do prefer verified academic domains versus gmail or other public mail accounts.

**Comment by volkankoc** [comment]
hi @samus-aran 
Thanks for the feedback. I filled in the required fields. But there is a problem: the information on my ID is written in Turkish. Also, the name of the school is not written on the ID, but the Ministry of National Education. Therefore, the system does not accept it. What can I do? I also have a document in my hand that shows that I am a teacher with a QR code approval from the Republic of Turkey e-government website.

**Comment by volkankoc** [comment]
Please help @samus-aran @mgriffin @zackfern @martinwoodward

**Reply by samus-aran** [reply]
Hi @volkankoc, again 😄 

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community.

Now let's get back to the original question. 🤔 

> But there is a problem: the information on my ID is written in Turkish

Ok, so here are some tips based on your latest submission:

- Use live image capture, not a screenshot. 
- Include both your student ID and previous evidence (that includes your school's name). It shouldn't matter that it's in Turkish 👍 
- We ask for an academic email because we've had students submit using ones before, this would be preferred. Not essential but would certainly help.

**Comment by volkankoc** [comment]
I couldn't get it accepted in any way. I uploaded a document with an electronic signature but it didn't work. I added a picture from inside the classroom but it didn't work. It accepted that I was a teacher for 2 years. It doesn't accept it anymore. I connect from the location where the school is but it still doesn't accept it. What should I do?

**Comment by volkankoc** [comment]
https://github.com/orgs/community/discussions/154120
Another friend has expressed his problems under this topic. Unfortunately, I am experiencing the same problems. Although I am a teacher, the approval is not coming. I am on campus and although I added a picture from inside the school classroom, it still does not work. I would be happy if you could find a solution.
